### PR TITLE
Create a test for display of authentication errors

### LINKED ISSUE #560: LoginPanel: test for display of authentication errors
` __tests__/components/LoginPanel.test.js`: test for display of authentication errors

e.g., test that the component properly displays whatever error message the sign-in or sign-out method sends back.

spawned by #528

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #560

### CODE REVIEW

File: __tests__/components/LoginPanel.test.js [hunk 1]
```diff
@@ -9,8 +9,7 @@ import { CognitoUserSession, CognitoAccessToken, CognitoIdToken, CognitoRefreshT
 
 
 describe('<LoginPanel /> when the user is not authenticated', () => {
-
-  let wrapper = shallow(<LoginPanel.WrappedComponent />)
+  const wrapper = shallow(<LoginPanel.WrappedComponent />)
```
Comment (mjgiarlo) [inline __tests__/components/LoginPanel.test.js]: :clap:

**Comment by ndushay** [comment]
In speaking with John yesterday, he apparently meant for two cases to be covered.  This one (the equivalent of "bad password") and 'user tried to sign out but their refresh token is bad' ... which I believe is currently doing a console.error approach but should instead show to user(?).

I took a run at this simple test and did not find the simple elegance of JC's approach here (though I do get points for similarly adding an id attribute to the element in question for testing) - my js / jest mocking skills need some shoring up.

**Comment by ndushay** [comment]
@jmartin-sul see prev comment - should there be another issue or just a PR or ??

**Comment by jmartin-sul** [comment]
@ndushay, just filed #594 ("better error messaging for failed signout")

**Pull request merged**